README.md

```diff
@@ -2,109 +2,146 @@
 
-To start or update the DL Workbench, run the starting script with a single-line command or manually.
-Two sections below describe the both options.
+#### Step 1. Set Up Python Virtual Environment
 
-> **IMPORTANT**: To download the script using the command line, you must have GNU Wget or cURL installed.
+To avoid dependency conflicts, use a virtual environment. Skip this step only if you do want to install all dependencies globally.
 
-#### <a id="pull">Pull the highest available Docker image of the DL Workbench</a>
+Create virtual environment by executing the following commands in your terminal:
 
-`docker pull openvino/workbench:latest`
-
-#### <a id="single-line-start">Single-Line Start</a>
+* On Linux and MacOS:
+```
+python3 -m pip install --user virtualenv
+python3 -m venv venv
+```
+* On Windows:
+```
+py -m pip install --user virtualenv
+py -m venv venv
+```
+#### Step 2. Activate Virtual Environment
 
-Choose one of the options:
-* Single-line command with GNU Wget:
-```bash
-wget https://raw.githubusercontent.com/openvinotoolkit/workbench_aux/master/start_workbench.sh && bash start_workbench.sh -IMAGE_NAME openvino/workbench -TAG latest
+* On Linux and MacOS:
+```
+source venv/bin/activate
+```
+* On Windows:
 ```
+venv\Scripts\activate
+```
+
+#### Step 3. Update PIP to the Latest Version
+Run the command below:
 
-* Single-line command with cURL:
-```bash
-curl -o start_workbench.sh https://raw.githubusercontent.com/openvinotoolkit/workbench_aux/master/start_workbench.sh && bash start_workbench.sh -IMAGE_NAME openvino/workbench -TAG latest
 ```
+python -m pip install --upgrade pip
+```
+#### Step 4. Install the Python Wrapper
+```
+pip install -U openvino-workbench
+```
+#### Step 5. Verify the Installation
 
-> **NOTE**: Execution of any of the above commands starts the DL Workbench in the interactive mode
-> with CPU support only. Go to the [List of All Arguments](./docs/Install_DL_Workbench.md#args) for other options.
+To verify that the package is properly installed, run the command below:
+```
+openvino-workbench --help
+```
+You will see the help message for the starting package if installation finished successfully.
 
-#### <a id="manual-start">Step-by-Step Start</a>
+#### Use the DL Workbench Starter
 
-1. Choose one of the options to download the script:
-   * Download with GNU Wget:
-   ```bash
-   wget https://raw.githubusercontent.com/openvinotoolkit/workbench_aux/master/start_workbench.sh
-   ```
-   * Download with cURL:
-   ```bash
-   curl -o start_workbench.sh https://raw.githubusercontent.com/openvinotoolkit/workbench_aux/master/start_workbench.sh
-   ```
+To start the latest available version of the DL Workbench, execute the following command:
 
-2. Open a terminal in the directory with the downloaded script and run the following command to make
-   the file executable:
-```bash
-chmod +x start_workbench.sh
+```
+openvino-workbench --image openvino/workbench:2021.3
```

Comment:
2021.3 or 2021.4?

Reply:
It will be updated to `21.4` once it is released.